packages/components/page-header/src/page-header.ts

```diff
@@ -0,0 +1,21 @@
+import type { ExtractPropTypes } from 'vue'
+
+export const pageHeaderProps = {
+  icon: {
+    type: String,
+    default: 'el-icon-back',
+  },
+  title: {
+    type: String,
+  },
```

Comment:
Use `title: String`, since no extra descriptor for this prop we should consider not using object notation.

Reply:
Done.